### DISCUSSION TITLE
Most categories not represented in the box score

### DISCUSSION THREAD

**Discussion by vwsong** [cwendt94/espn-api#407] [ESPN API]
Hi, saw that you added categories box score in this PR: https://github.com/cwendt94/espn-api/pull/391
So now we have points and categories accounted for, but I'm in a league where it's "most categories". Basically category scoring but 1 win 1 loss or 1 tie instead of categories. Currently when I pull the box score, I'm getting H2HPointsBoxScore because that's the default box score. 

Haven't looked too deep into it, but I'm thinking just adding it to the map should do the trick b/c the box score is still the same as categories. 

in box_score.py:
```
# helper function to get correct box score class
ScoringType = {'H2H_POINTS': H2HPointsBoxScore, 'H2H_CATEGORY': H2HCategoryBoxScore, `H2H_MOST_CATEGORIES`: H2HCategoryBoxScore}
get_box_scoring_type_class = lambda scoring_type: ScoringType.get(scoring_type, H2HPointsBoxScore)
```

You can use my league ID (should be public?) for testing: 1748254402

**Comment by cwendt94** [comment]
Awesome, I was not sure if H2H categories could be used for most categories or not! I will try it now!
